api/v01/entities/vrp_input.rb

```diff
@@ -157,7 +157,6 @@ module VrpConfiguration
     optional(:partition_metric, type: Symbol, documentation: { hidden: true }, desc: '[ DEPRECATED : use partitions structure instead ]')
     optional(:kmeans_centroids, type: Array[Integer], documentation: { hidden: true }, desc: '[ DEPRECATED : use partitions structure instead ]')
     optional(:cluster_threshold, type: Float, desc: 'Regroup close points which constitute a cluster into a single geo-located point')
-    optional(:force_cluster, type: Boolean, desc: 'Force to cluster visits even if containing timewindows and quantities')
```

Comment:
I think this is a very good idea because i) we don't really support it ii) it can lead to infeasible solutions and iii) no one uses it... However, we still need to do it the right way :/

Can you mark it "Deprecated" and "Hidden: true" here and then in validate_data.rb add a condition  to the `check_configuration` function that raises and `UnsupportedProblemError` if force_cluster is true. Otherwise, we silently ignore a deprecated parameter :see_no_evil: :hear_no_evil: :speak_no_evil:

Reply:
I mean, I get you but it was in our documentation since 2016 and since "we" are changing the behaviour, we need to advertise the change properly. 

If a user (this could even be you) has an old JSON file with `force_cluster: true` we need to be able to refuse that file since force clustering was once supported but it is not supported anymore. Otherwise, we have no way to check since declared_params remove unknown fields -- and for obvious reasons, we cannot refuse all undeclared params -- and the user has no way to understand the clustering is not done.

Reply:
Raising an error for field which have no more purpose seems a bit dangerous for me. It it was a parameter used by a user in production, we would not block the solve. What if i only generate a warning sentry on our side ?

Reply:
I mean yeah, warning sentry would work too :+1: 

To be honest, I think completely ignoring an option is far more dangerous than blocking the user -- it is us changing the behaviour, we need to properly communicate it -- so I don't understand exactly why you are okay with the first but not with the latter. 

Anyways, in the sentry warning, can you mention that this parameter is deleted and we need to explain to the user that it is ignored and they should stop sending it.